### ISSUE TITLE
Make this repo easier to find via esbench.com

### ISSUE THREAD

**Issue by tivac** [developit/esbench#5]
Maybe via an "About" page or something similar?

**Comment by developit** [comment]
Indeed. Also I just really need to open source esbench. It's not open source right now because the code is less than stellar :P

**Comment by NoNameProvided** [comment]
I think you should not worry about the quality of the code, you have wrote it and that is what matters. Community can help you raising the quality. :) So +1 for open sourcing because this project is really cool and can be the successor of jsperf, because jsperf looks very dead now.

**Comment by developit** [comment]
@NoNameProvided Point taken :)

I've got a couple little minor things I need to do (call them scrubbing) before I could publish it, but I guess I could just make it public after that.
